weave-daemonset.yaml

```diff
@@ -48,8 +50,6 @@ spec:
               cpu: 200m
             requests:
               cpu: 25m
-          securityContext:
-            privileged: true
```

Comment:
It is required, but this is the second time it appears in this file.  I should perhaps have done that in a separate commit.